src/components/pick-list-item/pick-list-item.tsx

```diff
@@ -62,7 +62,7 @@ export class PickListItem implements ConditionalSlotComponent, InteractiveCompon
   @Prop({ reflect: true }) nonInteractive = false;
 
   /**
-   * Determines the icon SVG symbol that will be shown. Options are circle, square, grip or null.
+   * Determines the icon SVG symbol that will be shown. Options are `"circle"`, `"square"`, `"grip"` or `null`.
```

Comment:
@macandcheese is there a reason why we have a see jsdoc tag here? It looks like the types are set on the property

```ts
 @Prop({ reflect: true }) icon?: ICON_TYPES | null = null;
```

Reply:
https://developers.arcgis.com/calcite-design-system/components/pick-list-item/#component-api-properties-icon

I think because they are stored in an external file they don’t end up in the jsdoc. At least that was the case awhile ago. If that isn’t the case anymore we can remove the ‘@see’

Reply:
Based on the conversation above, going to leave the values in the docs, so folks can discern the values with/without accessing the "View documentation" link.